soc/v2/nuvoton/npcx/soc.yml

```diff
@@ -0,0 +1,16 @@
+series:
```

Comment:
should we have a family: `ncpx`  described in this file ?

Reply:
fixed